content/swiftui/concepts/widgets/widgets.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: 'SwiftUI Widgets'
+Description: 'Building user interfaces across Apple platforms.'
+Subjects: 
+    - 'iOS Development'
+Tags: 
+    - 'SwiftUI'
+    - 'User Interface'
+    - 'iOS'
+CatalogContent: 
+    - 'paths/iOS Development'
+    - 'Learn SwiftUI'
```

Comment:
```suggestion
    - 'learn-swift'
```

Reply:
I don't believe there's a learn swiftui course, but do link me if this isn't accurate